src/Core/Framework/Store/Struct/ExtensionStruct.php

```diff
@@ -133,10 +133,23 @@ class ExtensionStruct extends Struct
      */
     protected string $updateSource = self::SOURCE_LOCAL;
 
-    protected bool $allowDisable = true;
```

Comment:
i think removing this prop was not intendend? :thinking:

Reply:
:O No it wasn't. Classical "I fix in vendor folder and copy it over here"-issue

Reply:
> :O No it wasn't. Classical "I fix in vendor folder and copy it over here"-issue

Could be me 😆